jpugdoc/conflict.go

```
@@ -2,26 +2,75 @@ package jpugdoc
-	for _, catalog := range notEq {
-		fmt.Println(gchalk.Green(catalog.en))
-		fmt.Println(catalog.ja)
-		fmt.Println()
+	return writeConflict(os.Stdout, findNotSameCommon(seen), opt)
+}
+
+func writeConflict(w io.Writer, catalogs []conflictCatalog, opt ConflictOption) error {
+	switch opt.Format {
+	case ConflictFormatText:
+		for _, catalog := range catalogs {
+			fmt.Fprintln(w, gchalk.Green(catalog.en))
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)